src/hierarchical_deterministic/bip39/mnemonic_uniffi_fn.rs

```diff
@@ -1,5 +1,30 @@
 use crate::prelude::*;
 
+/// Returns new mnemonic from a string of words
+#[uniffi::export]
+pub fn new_mnemonic_from_phrase(phrase: String) -> Result<Mnemonic> {
+    Mnemonic::from_phrase(&phrase.to_owned())
```

Comment:
&string.to_owned()

Does nothing, I think you should remove to_owned.

If you run clippy fix it should fix this?